src/jit/gtlist.h

```diff
@@ -173,6 +173,8 @@ GTNODE(SUB_LO     , "-Lo"          ,0,GTK_BINOP)
 GTNODE(SUB_HI     , "-Hi"          ,0,GTK_BINOP)
 GTNODE(DIV_HI     , "/Hi"          ,0,GTK_BINOP)
 GTNODE(MOD_HI     , "%Hi"          ,0,GTK_BINOP)
+GTNODE(LSH_HI     , "<<Hi"         ,0,GTK_BINOP)
+GTNODE(RSH_LO     , ">>Lo"         ,0,GTK_BINOP)
```

Comment:
These shift hi/lo opers are different from the rest that these take a long type operand as op1.  It would be a good idea to add a couple of comment lines explaining their purpose and where they are used.